test/gaugeTest.js

```diff
@@ -13,7 +13,7 @@ describe('gauge', () => {
 		});
 		describe('with a parameter for each variable', () => {
```

Comment:
I think this whole `describe` can be removed, the tests are duplicated but with different arguments. See the `describe('with parameters as object', ...` (https://github.com/siimon/prom-client/blob/e9bad9cb57f8dcf396a95d4f1e7442f4f3d09f97/test/gaugeTest.js#L216)

It is the same for all the tests for the different metric types.